person.rb

```diff
@@ -27,4 +28,9 @@ def can_use_services?
   def correct_name
     @name
   end
+
+  def add_rental(rental)
+    @rental.push(rental)
+    rental.person = self
+  end
```

Comment:
- [ ] Great job creating a method for adding rentals but it should look like this:

```ruby
 def add_rental(book, date)
    # *The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association
    Rental.new(date, book, self)
  end
```